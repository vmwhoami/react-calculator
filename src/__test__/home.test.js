import React from 'react';
import renderer from 'react-test-renderer';


it('home renders correctly', () => {
  const tree = renderer
    .create(
      <main className="maincont">
        <article>
          <h1>Welcome to our page!</h1>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit.
            Earum, consequatur deserunt sed temporibus optio illo dolorum
            aperiam, repellat ipsum est omnis dolores iure molestiae voluptas
            perferendis ex officia! Porro, deserunt.
        </p>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit.
            Earum, consequatur deserunt sed temporibus optio illo dolorum
            aperiam, repellat ipsum est omnis dolores iure molestiae voluptas
            perferendis ex officia! Porro, deserunt.
        </p>
        </article>
      </main>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

