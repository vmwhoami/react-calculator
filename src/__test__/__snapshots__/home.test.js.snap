// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home renders correctly 1`] = `
<main
  className="maincont"
>
  <article>
    <h1>
      Welcome to our page!
    </h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum, consequatur deserunt sed temporibus optio illo dolorum aperiam, repellat ipsum est omnis dolores iure molestiae voluptas perferendis ex officia! Porro, deserunt.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum, consequatur deserunt sed temporibus optio illo dolorum aperiam, repellat ipsum est omnis dolores iure molestiae voluptas perferendis ex officia! Porro, deserunt.
    </p>
  </article>
</main>
`;
